src/rgw/rgw_civetweb.cc

```diff
@@ -111,7 +111,7 @@ void RGWCivetWeb::init_env(CephContext *cct)
   }
 
   env.set("REQUEST_METHOD", info->request_method);
-  env.set("REQUEST_URI", info->uri);
+  env.set("REQUEST_URI", info->request_uri); // get the full uri, we anyway handle abs uris later
```

Comment:
https://github.com/civetweb/civetweb/blob/59fa55a87cdc8397f4f88e4b76881857f6e0622e/docs/api/mg_request_info.md civetweb marked their uri field as deprecated and introduced local_uri and request_uri, and uri field was set as local_uri for now, but maybe removed in future